automatic/kitty/update.ps1

```diff
@@ -1,7 +1,6 @@
 import-module au
 
-$releases = 'https://github.com/cyd01/KiTTY/releases/latest'
-$padVersionUnder = '0.71.1'
+$latestRelease = 'https://api.github.com/repos/cyd01/KiTTY/releases/latest'
 
 function global:au_SearchReplace {
   @{ }
```

Comment:
Please put in this function what went into au_BeforeUpdate as per usual. Remove table from VERIFICATION.txt and use standard form.